ocs_ci/ocs/constants.py

```diff
@@ -1095,6 +1095,9 @@
 
 PRODUCTION_JOBS_PREFIX = ["jnk"]
 
+# Cloud Manager available platforms
+cld_mgr_platforms = ["aws", "gcp", "azure", "ibmcos"]
```

Comment:
We already have a `CLOUD_PLATFORMS` constant. Should we just add `ibmcos` to this?

https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py#L590

If we need two separate constants, this one should be converted to all caps.

Reply:
Theoretically, we could add IBMCOS to the existing constant, but I suspect it might become problematic in the future, when we might have to support S3 clients that we don't necessarily deploy on. It'll become confusing.

Thus, I opted to keep the constant, and convert it to all caps.